### DISCUSSION TITLE
[remark] Typescript definitions?

### DISCUSSION THREAD

**Discussion by AlbinoGeek** [remarkjs/remark#828] [Q&A] [answered]
So, yarn's telling me there's no `@types/remark`, nor `@DefinitelyTyped/remark` -- only `paprist/remark-types` at best -- is there any hint towards what typings package is supported by the project maintainers, if any?

**Comment by ChristianMurphy** [comment, marked as answer]
`remark` version 11 and higher include typings directly in the `remark` package.

```
npm install remark
```

**Reply by AlbinoGeek** [reply]
Gotcha! VSCode just upset again I suppose.
